attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
And if we take that out, then the default is here... https://github.com/chef-cookbooks/omnibus/blob/master/attributes/default.rb#L43